src/utils/rest.ts

```diff
@@ -1,5 +1,6 @@
-import fetch, { RequestInit } from 'node-fetch';
+import { RequestInit } from 'node-fetch';
```

Comment:
This can be updated to `import type {...` now that an async shim is being used instead

Reply:
i did attempt to change type imports with the following in the tsconfig.json...

```json
...
    "importsNotUsedAsValues": "error",
...
```

however, it seems like your current linting set up does not support this syntax :(

Reply:
We can look at this as a follow-up on our side, in that case!